### PR TITLE
curve/tool fs usage metadata

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

```
curve fs usage metadata
+--------------------+--------+--------+--------+
|   metaserverAddr   | total  |  used  |  left  |
+--------------------+--------+--------+--------+
| 10.219.192.50:6800 | 2.2 TB | 192 GB | 2.0 TB |
| 10.219.192.50:6802 | 2.2 TB | 192 GB | 2.0 TB |
| 10.219.192.50:6801 | 2.2 TB | 192 GB | 2.0 TB |
+--------------------+--------+--------+--------+


curve fs usage metadata  -f json | jq
{
"error": {
"code": 0,
"message": "success"
},
"result": {
"metadataUsages": [
{
"metaserverAddr": "10.219.192.50:6800",
"total": "2199023255552",
"uint": "Byte",
"used": "192220069888"
},
{
"metaserverAddr": "10.219.192.50:6802",
"total": "2199023255552",
"uint": "Byte",
"used": "192220069888"
},
{
"metaserverAddr": "10.219.192.50:6801",
"total": "2199023255552",
"uint": "Byte",
"used": "192220069888"
}
]
}
}
```
image
image

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: tools-v2/pkg/cli/command/curvefs/list/mountpoint/mountpoint.go [hunk 1]
```diff
@@ -100,6 +100,13 @@ func updateTable(table *table.Table, info *mds.ListClusterFsInfoResponse) {
 	fssInfo := info.GetFsInfo()
 	rows := make([]map[string]string, 0)
 	for _, fsInfo := range fssInfo {
+		if len(fsInfo.GetMountpoints()) == 0 {
+			row := make(map[string]string)
+			row["fs id"] = fmt.Sprintf("%d", fsInfo.GetFsId())
```
Comment (Wine93) [inline tools-v2/pkg/cli/command/curvefs/list/mountpoint/mountpoint.go:105]: maybe you can use const string for row filed.

File: tools-v2/pkg/cli/command/curvefs/usage/metadata/metadata.go [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/curvefs/proto/topology"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+)
+
+type MetadataRpc struct {
+	Info      basecmd.Rpc
```
Comment (Wine93) [inline tools-v2/pkg/cli/command/curvefs/usage/metadata/metadata.go:42]: code style.

Comment (Cyber-SiKu) [reply tools-v2/pkg/cli/command/curvefs/usage/metadata/metadata.go:42]: > code style.

fix

File: tools-v2/pkg/cli/command/curvefs/list/mountpoint/mountpoint.go (hunk 1 above)
Comment (Cyber-SiKu) [reply tools-v2/pkg/cli/command/curvefs/list/mountpoint/mountpoint.go:105]: > maybe you can use const string for row filed.

After this, I will submit a pr to modify it. There is also this problem later pr.

**Review by Wine93 — APPROVED** [review]

**Review by aspirer — APPROVED** [review]

**Pull request merged**